devsetup/tripleo/ceph.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
+#
+# Copyright 2023 Red Hat Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+set -ex
+
+# generate inventory based on config-download.yaml
+mkdir -p $HOME/overcloud-deploy/overcloud/
+export INV=$HOME/overcloud-deploy/overcloud/tripleo-ansible-inventory.yaml
+python /tmp/generate_ceph_inventory.py -d config-download.yaml -i $INV
+
+# in CI we'll copy ci-framework from the controller node, so we can pickup
+# depends-on, but this block makes sure we have ci-framework in case this runs
+# outside
+if [[ ! -d "$HOME/ci-framework" ]]; then
+    git clone https://github.com/openstack-k8s-operators/ci-framework.git
+fi
+
+cd ci-framework
+# create block devices on all compute nodes
+ansible-playbook -i $INV playbooks/ceph.yml --tags block -e num_osds=3
+cd ..
+
+# TODO jgilaber, will need to reduce the amount of block devices when we
```

Comment:
It's OK to have 9 OSDs if you have 3 computes. You could reduce it later if you need to for some reason, but it's not an essential todo.

Reply:
ack, I saw the comment in your README and though it was necessary, I'll remove it in my next update